lukasz-lapczyk/review-dealer
# Add logo image to company profile

# Overview
Users can add logo images to their companies' profiles. Logo file must be in jpg/jpeg/png format and less than 3MB in size. 

# Testing prerequisites
- bundle install
- db:migrate
- db:seed

# Testing
- navigate to root directory
- log in with 'user_3@example.org', pass: Secret39 or create new account
- click 'Create company' on navbar
- fill in form with valid name and choose valid image file
- [ ] company should be created
- click on your new company name on companies listing or 'My company' link on navbar
- [ ] image should be displayed under company name (reduced in size - max 50x50 px)
